File: src/main/scala/org/xbery/artbeams/products/domain/EditedProduct.scala
package org.xbery.artbeams.products.domain

import net.formio.binding.ArgumentName

/**
  * @author Radek Beran
  */
case class EditedProduct(
  @ArgumentName("id")
  id: String,
  @ArgumentName("slug")
  slug: String,
  @ArgumentName("title")
  title: String,
  @ArgumentName("fileName")
  fileName: Option[String]
)

File: src/main/kotlin/org/xbery/artbeams/products/domain/EditedProduct.kt
package org.xbery.artbeams.products.domain

import java.math.BigDecimal

/**
 * @author Radek Beran
 */
data class EditedProduct(
    val id: String,
    val slug: String,
    val title: String,
    val subtitle: String?,
    val fileName: String?,
    val listingImage: String?,
    val image: String?,
    val confirmationMailingGroupId: String?,
    val mailingGroupId: String?,
    val priceRegularAmount: BigDecimal?,
    val priceDiscountedAmount: BigDecimal?,
    val simpleShopProductId: String?
)
